collections/_api_standards/3-api-iterating.md

```diff
@@ -0,0 +1,60 @@
+
+### Specifying a version
+
+There are multiple ways of specifying versioning, and there is no right or wrong answer.  Common approaches of versioning APIs are given in the table below.
+
+|Versioning approach|Example|Notes|
+|---|---|---|
+|URI path|`GET /api/v2/users`|Easy to implement, but... implies different resources (as opposed to version of a resource), is more work for client to upgrade, and versions the entire API in one go (rather than offering a more nuanced approach)|
+|URI query string|`GET /api/users?v=2`|Does not seem a natural or RESTful way of versioning|
+|Custom header|`GET /api/users`<br />`Content-Type: application/json`<br />`X-Version: 2`|Introducing a new header (which is not part of core the HTTP standard) does not seem logical.|
+|Media Type|`GET /api/users`<br />`Accept: application/json;version=2`<br /><br />`POST /api/users`<br />`Content-Type: application/json;version=2`<br />`Content-Length:12`<br />`Hello there!`|A core part the HTTP approach, and RESTful in its nature.  This is our preferred approach.  This strategy can make the API harder to consume for simple script clients, but most AS APIs are NOT consumed from simple scripting languages (thus, this is less of an issue).|
+
+### Media Type versioning in .netcore
+
+.netcore provides API versioning out of the box.  Simply add the the [Microsoft.AspNetCore.Mvc.Versioning]( https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Versioning/) package to your API project.  The package supports a variety of versioning strategies, [and support is provided for versioning by media-type](https://github.com/microsoft/aspnet-api-versioning/wiki/Versioning-by-Media-Type).  The .netcore versioning approach can be used to version individual actions or entire controllers.  The approach is suitable for scenarios where version changes are of simple to medium complexity.
+
+In some cases, a new API version will be significantly different to its predecessor, and maintaining the two side-by-side in the same deployed code base may be an issue.  In these cases, it will be necessary to implement an alternative approach, potentially using a policy within APIM to route requests to different deployed versions of the API.  In such cases, the delivery team are advised to liaise with DevOps and the Quality Guild in order to agree an appropriate approach.

```

Comment:
and architects...or maybe archs instead of quality guild. I want to make it a bit more accessible to the teams. The archs should then be responsible for validating it with the guild.

Reply:
As requested, changes made to make it clear delivery team engage with architects re APIs (as opposed to quality guild).